packages/server/modules/comments/index.ts

```diff
@@ -17,9 +17,12 @@ const commentsModule: SpeckleModule = {
       const notifyUsersOnCommentEvents = notifyUsersOnCommentEventsFactory({
         commentsEventsListen: CommentsEmitter.listen,
         publish: publishNotification,
-        addStreamCommentMentionActivity: addStreamCommentMentionActivityFactory({
-          saveActivity: saveActivityFactory({ db })
-        })
+        addStreamCommentMentionActivityResolver: async ({ streamId }) => {
+          const projectDb = await getProjectDbClient({ projectId: streamId })
+          return addStreamCommentMentionActivityFactory({
+            saveActivity: saveActivityFactory({ db: projectDb })
```

Comment:
activities are currently not saved to the projectDb

it can be considered as non project data, rather an audit trail. In that sense, having a consistent timeline is important, so we cannot shard the data easily to multiple DBs